cli/command/system/completion.go

```diff
@@ -0,0 +1,99 @@
+package system
+
+import (
+	"strings"
+
+	"github.com/docker/cli/cli/command/completion"
+	"github.com/spf13/cobra"
+)
+
+var (
+	eventFilters = []string{"container", "daemon", "event", "image", "label", "network", "node", "scope", "type", "volume"}
+	eventNames   = []string{
```

Comment:
This would be added in the moby codebase?

Reply:
Yes, correct; we vendor those types from the Moby repo.

Converting to a string (or slice of strings) would then probably be left to the consumer (so that's something we can do in this repository).

Reply:
Great. I'll make a suggestion. This might take some time, though.
Thanks for your support.

Reply:
@thaJeztah I added completions for the remaining filters.
At that stage, the `completeFilters` was way to long and ugly, so I decided to introduce some helper functions.
These helper functions simplify error handling very much, as they just return empty arrays in case of API errors.
PTAL, IMHO the PR should be complete now.